extensions/uuid-generator/package.json

```diff
@@ -160,19 +174,23 @@
     }
   ],
   "dependencies": {
-    "@raycast/api": "^1.55.0",
-    "typeid-js": "^1.1.0",
-    "ulidx": "^2.0.0",
-    "uuid": "^10.0.0"
+    "@raycast/api": "^1.91.2",
+    "typeid-js": "^1.2.0",
+    "ulidx": "^2.4.1",
+    "uuid": "^11.0.5"
   },
   "devDependencies": {
-    "@raycast/eslint-config": "^1.0.6",
-    "@types/node": "^20.4.2",
+    "@eslint/compat": "^1.2.6",
+    "@eslint/eslintrc": "^3.2.0",
+    "@eslint/js": "^9.20.0",
+    "@raycast/eslint-config": "^1.0.11",
+    "@types/node": "^22.13.4",
     "@types/uuid": "^10.0.0",
-    "eslint": "^8.45.0",
-    "eslint-plugin-import": "^2.27.5",
-    "react-devtools": "^4.28.0",
-    "typescript": "^5.1.6"
+    "@typescript-eslint/eslint-plugin": "^8.24.0",
+    "eslint": "^9.20.1",
+    "eslint-plugin-import": "^2.31.0",
```

Comment:
Why?

Reply:
Just trying to keep packages up to date to avoid weird old compatibility issues

Reply:
AFAIK, Raycast does not support ESLint 9 yet.

Reply:
ah, interesting, that's why this was harder than I expected :)